hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -78,6 +100,7 @@ def lambda_handler(event, context):
                 )
```

Comment:
I'm realizing now the this version of the matcher wont work with them multi-indexer in PR #460. That version merges multiple lists of (hash, metadata) tuples into a unique set of hashes with merged metatdata. Metadata has a list of privacy groups indicating which collaboration it came from. 

I'll put up a fix. I think it could be either here or in pdq_indexer and I'm not sure which will be more efficient. Do you feel strongly?

Reply:
Hmm, in my mind these were compatible, but I have to read more to figure out what the break is. See the comment about the division of labor above.